nginx-router/docker/confd/templates/app.conf.tmpl

```diff
@@ -26,7 +27,15 @@ server {
     proxy_buffering off;
 
     location / {
-        proxy_pass {{ getv (printf "/opg/nginx/router/%s/target" .) }};
+        {{ with $force_dns_resolution := (printf "/opg/nginx/router/%s/force/dns/resolution" $vhost_id) }}
```

Comment:
as you are using $force_dns_resolution only once then with doesn't make sense - to keep in sync with coding standard in previous lines